draft-irtf-qirg-principles-03.xml

```diff
@@ -795,6 +820,47 @@
            information via separate classical channels which the repeaters will
            have to correlate with the qubits stored in their memory.</t>
 
```

Comment:
This title is confusing. Maybe something simpler? E.g. "First generations of quantum networks will not be store-and-forward" perhaps?

Reply:
"First generations of quantum networks will not be store-and-forward" is fine.